scripts/crossref_doi_retrieval.ipynb

```diff
@@ -73,22 +73,34 @@
     "    if not 1800 <= row.year <= 2021:\n",
     "        return None\n",
     "    year = str(int(row.year))\n",
-    "    url = 'https://api.crossref.org/works/?query.title=' + urllib.parse.quote(row.title) + \\\n",
+    "\n",
+    "    try: \n",
+    "        title = urllib.parse.quote(row.title)\n",
+    "    except:\n",
+    "        print(\"Title could not be encoded\")\n",
```

Comment:
here it would be nice to increment the counter of encoding-failed titles

Reply:
Fully agree